ios-base/DroidKaigi 2020/Session/Views/SessionPageViewController.swift

```diff
@@ -0,0 +1,103 @@
+import UIKit
+
+enum SessionViewControllerType: Int {
+    case day1 = 1
+    case day2 = 2
+    case myPlan = 3
+
+    var date: Date? {
+        let calendar = Calendar(identifier: .gregorian)
+        switch self {
+        case .day1:
+            return calendar.date(from: .init(year: 2020, month: 2, day: 20))
+        case .day2:
+            return calendar.date(from: .init(year: 2020, month: 2, day: 20))
```

Comment:
```suggestion
            return calendar.date(from: .init(year: 2020, month: 2, day: 21))
```

Reply:
Oh... Thank you!
I will take this suggested changes.